src/apns.erl

```diff
@@ -270,18 +273,18 @@ default_connection() ->
         get_env(timeout, DefaultConn#apns_connection.timeout)
     , error_fun =
         case get_env(error_fun, DefaultConn#apns_connection.error_fun) of
-          {M, F} -> fun(I, S) -> M:F(I, S) end;
-          Other -> Other
+                                                  {M, F} -> fun(I, S, Msg) -> M:F(I, S, Msg) end;
```

Comment:
Line 276 is too long:                                                   {M, F} -> fun(I, S, Msg) -> M:F(I, S, Msg) end;.